.env.example

```diff
@@ -11,3 +11,10 @@ REACT_APP_SENTRY_DSN=
 
 # Wallet Connect project ID
 REACT_APP_PROJECT_ID=e7e6e90622ec8e65c4d02e829a68529a
+
+# The RPC URL will be visible in the browser network tab (leaking the API key). It is advised to whitelist only API3 DAO
+# contracts making the RPC URL useless for others. To support Wallet Connect with MM wallet without errors, the
+# following multicall contract needs to be whitelisted as well:
+#
+# https://etherscan.io/address/0xca11bde05977b3631167028862be2a173976ca11
```

Comment:
May as well link this https://github.com/wagmi-dev/viem/blob/main/src/chains/definitions/mainnet.ts